packages/textInput/components/TextInput.tsx

```diff
@@ -7,12 +7,8 @@ import {
   getInputAppearanceStyle,
   inputContainer
 } from "../../shared/styles/formStyles";
-import {
-  flex,
-  flexItem,
-  inputReset,
-  padding
-} from "../../shared/styles/styleUtils";
+import { inputReset, padding } from "../../shared/styles/styleUtils";
+import { flex, flexItem } from "../../shared/styles/styleUtils/layout/flexbox";
```

Comment:
Enzyme would fail with `flex is not a function` when I imported directly from styleUtils and used the `mount` function. No idea why.